codebuild/common-linux.sh

```diff
@@ -13,4 +13,4 @@ ENDPOINT=$(aws secretsmanager get-secret-value --secret-id "unit-test/endpoint"
 cd $CODEBUILD_SRC_DIR
 
 ulimit -c unlimited
-mvn -B test -DredirectTestOutputToFile=true -DreuseForks=false -Dendpoint=$ENDPOINT -Dcertificate=/tmp/certificate.pem -Dprivatekey=/tmp/privatekey.pem -Drootca=/tmp/AmazonRootCA1.pem
+mvn -B test -DredirectTestOutputToFile=true -DreuseForks=false -Dendpoint=$ENDPOINT -Dcertificate=/tmp/certificate.pem -Dprivatekey=/tmp/privatekey.pem -Drootca=/tmp/AmazonRootCA1.pem -Daws.crt.debugnative=true
```

Comment:
Do you mean to check this in?

Reply:
Yeah, it makes the resource count assertions actually do something in the tests